src/objects/Error_Settings__c.object

```diff
@@ -69,6 +69,16 @@ To view these jobs, search for Scheduled Jobs in Salesforce Setup.</description>
         <trackTrending>false</trackTrending>
         <type>TextArea</type>
     </fields>
+    <fields>
+        <fullName>OverrideFeature_PilotEnabled__c</fullName>
+        <defaultValue>false</defaultValue>
+        <description>Do Not Update. For internal Testing Use Only. Allows limited access override of the PilotEnabled Feature Parameter in Scratch Orgs to support pilot testing in orgs where there is no associated Product License in the LMO.</description>
```

Comment:
@sam-knox Would you mind reviewing the label and helptext/description on this new field? It is strictly for our own internal use (at SFDO) and will not be rendered in the NPSP Settings UI or included in the documentation for customers.

Reply:
@force2b - Sure, how about this?

"Do Not Update. For Salesforce.org pilot testing only. Allows limited access override of the PilotEnabled Feature Parameter in Scratch Orgs to support pilot testing in orgs where there is no associated Product License in the LMO."

I think it might help to make it clear who the testing is for.